tests/Integration/Schema/Directives/CountDirectiveDBTest.php

```diff
@@ -420,4 +420,44 @@ public function testResolveRelationItemsAndCount(): void
             ],
         ]);
     }
+
+    public function testCountModelWithField(): void
+    {
+        $this->schema = /** @lang GraphQL */ '
+        type Query {
+            tasks: Int @count(model: "Task", field: "difficulty")
+        }
+        ';
+        factory(Task::class, 3)->create(['difficulty' => 1]);
+
+        $this->graphQL(/** @lang GraphQL */ '
+        {
+            tasks
+        }
+        ')->assertExactJson([
+            'data' => [
+                'tasks' => 3,
+            ],
+        ]);
+    }
+
+    public function testCountModelWithFieldAndDistinct(): void
+    {
+        $this->schema = /** @lang GraphQL */ '
+        type Query {
+            tasks: Int @count(model: "Task", distinct: true, field: "difficulty")
+        }
+        ';
+        factory(Task::class, 3)->create(['difficulty' => 1]);
```

Comment:
https://github.com/nuwave/lighthouse/blob/master/CONTRIBUTING.md#test-data-setup

Reply:
I need to create three tasks with the same difficulty in order to test the **field** and **distinct** combination, please let me know if there is a better way. 🙏🏼

Reply:
```suggestion
        foreach (factory(Task::class, 3)->make() as $task) {
            $task->difficulty = 1;
            $task->save();
        }
```

A bit more verbose, but typesafe and able to be refactored.